include/svnlook.php

```diff
@@ -812,8 +812,8 @@ function applyGeshi($path, $filename, $language, $rev, $peg = '', $return = fals
 		$this->geshi->set_tab_width($this->repConfig->getExpandTabsBy());
 
 		if ($highlight == 'file') {
-			$this->geshi->enable_line_numbers(GESHI_FANCY_LINE_NUMBERS,1);
-			$this->geshi->set_overall_id('line');
+			$this->geshi->enable_line_numbers(GESHI_FANCY_LINE_NUMBERS);
+			$this->geshi->set_overall_id('geshi');
```

Comment:
I don't see a better way without fixing geshi first. The issue that Geshi can be used multiple times in a PHP script, mean you would produce clashing its. That is why you need an overall id.
Bascially, they need to introduce a line number override and we make sure that only a single instance of geshi is used per svnlook. Which is.